_config.yml

```diff
@@ -11,7 +11,7 @@ github_username: Artsy
 disqus_short_name: artsy
 disqus_show_comment_count: false
 
-exclude: [vendor]
+exclude: [vendor, LICENSE, "_posts/LICENSE"]
```

Comment:
YAML is so weird. So you can mix strings wrapped with quotes with strings unwrapped in the same array? Maybe we should pick one style for this array:

```yaml
exclude: ["vendor/", "LICENSE", "_posts/LICENSE"]
```

Reply:
Sounds good to me 👍